tomcat/osmap.yaml

```diff
@@ -22,7 +22,7 @@ CentOS:
   native_pkg: tomcat-native
   manager_pkg: tomcat-admin-webapps
   main_config_template: salt://tomcat/files/tomcat-default-CentOS.template
-openSUSE:
+Suse:
```

Comment:
I wonder if we need to match on both here -- I'm 80% sure `openSUSE` was correct at some point in the past. Perhaps they've since changed it but I don't have quick access to old installs to check.

Reply:
Agreed. I'm share this opinion that it must have worked sometime on particular Suse and Salt versions - this is good observation for osmap.yaml files in general.